### PR TITLE
Rename Conf to File in PropertyNameKind #19

### LINKED ISSUE #19: Rename Conf in PropertyNameKind to File
The `PropertyNameKind` enum consists of three entries:
Conf(String),
Env,
Cli,

All of them offer config options. "Conf" in the enum refers to a config file (and file name via String).
For consistency we therefore should rename "Conf" to "File".

### PR DESCRIPTION
**Pull request by u5surf**
Fixes #19 

rename and replace Conf to File
update test data

### CODE REVIEW

File: src/types.rs [hunk 1]
```diff
@@ -46,15 +46,15 @@ impl fmt::Display for PropertyName {
...
+    File(String),
     Env,
     Cli,
 }
 
 impl PropertyNameKind {
     pub fn get_file_name(&self) -> String {
         match self {
-            PropertyNameKind::Conf(conf) => conf.clone(),
+            PropertyNameKind::File(conf) => conf.clone(),
```
Comment (maltesander) [inline src/types.rs:57]: Can we use name / file_name here? Conf does not make sense anymore.
`PropertyNameKind::File(name) => name.clone()`

**Review by maltesander — CHANGES_REQUESTED** [review]
Thank you very much for the contribution. Just one small comment. Looks good otherwise!

File: src/types.rs (hunk 1 above)
Comment (u5surf) [reply src/types.rs]: I fixed this.

**Review by maltesander — APPROVED** [review]
Approved. Thank you very much @u5surf !

**Pull request merged**